src/smt.cpp

```diff
@@ -406,9 +406,12 @@ bool Expr::isTrue() const {
 
 bool Expr::isVar() const {
   bool res = false;
-  IF_Z3_ENABLED(res |= z3 && z3->is_app() && z3->is_const() &&
-                       !z3->is_numeral());
-  // TODO: CVC5
+  IF_Z3_ENABLED(
+    if (z3) res |= z3->is_app() && z3->is_const() && !z3->is_numeral()
+  );
+  IF_CVC5_ENABLED(
+    if (cvc5) res |= !(cvc5->isIntegerValue() || cvc5->isBitVectorValue())
```

Comment:
We also have boolean constants.
BTW, is there a better way than this? We may have FPA constants at some point as well.

Reply:
I'll update to consider FPA and boolean constants as well.
I couldn't find a proper api to test whether the term is variable :(